.ci/docker/ci_commit_pins/executorch.txt

```diff
@@ -0,0 +1 @@
+9682172576d5d9a10f3162ad91e0a32b384a3b7c
```

Comment:
When this is a commit of a nightly build, we don't have to build ExecuTorch from source, but would do otherwise correct?